### ISSUE TITLE
Layout picker isn't accessible

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#143667]
Testing #143581

* use screen reader mode
* run F1 > Customize Layout
* the checked state of Side Bar Position and Panel Alignment aren't read out

<img width="1105" alt="Screen Shot 2022-02-23 at 09 16 55" src="https://user-images.githubusercontent.com/1794099/155282766-27bf411a-75b7-43f3-a3a0-22aa62ce32e1.png">

**Comment by TylerLeonhardt** [comment]
@sbatten are you saying it works with NVDA but not VoiceOver?

cc @isidorn

**Comment by sbatten** [comment]
@TylerLeonhardt that's right

**Comment by isidorn** [comment]
@TylerLeonhardt we should still have the ability internally for quick pick clients to define what roles their elements are.
In this particular case the role should be `checkbox` and not `listitem`.

**Comment by TylerLeonhardt** [comment]
@isidorn agreed. This is essentially like the API `AccessibilityInformation`. I would like to light that up.
